staging/provisioning/windows/cleanupnetwork.ps1

```diff
@@ -0,0 +1,67 @@
+$Global:ClusterConfiguration = ConvertFrom-Json ((Get-Content "c:\k\kubeclusterconfig.json" -ErrorAction Stop) | out-string)
```

Comment:
This script also deletes container instances so maybe maybe rename it to be more generic?

Reply:
removing the containers is part of the network clean up so I think it's fine

Reply:
works for me